charts/osac/templates/hooks/pre-install-validate.yaml

```
@@ -0,0 +1,102 @@
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "osac.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-weight": "-11"
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "osac.fullname" . }}-validate
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)